src/App.tsx

```diff
@@ -142,6 +142,10 @@ const App = () => {
 
 	const backgroundColor = urlParams.get('bg') || env.DESCOPE_BG_COLOR;
 
+	const storeLastAuthUser =
+		urlParams.get('storeLastAuthUser') === 'true' ||
```

Comment:
URL params are case insensitive , 

I would change it to `store-last-auth-user`